src/main/java/com/example/bookstore/repository/cartitem/CartItemRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.bookstore.repository.cartitem;
+
+import com.example.bookstore.model.CartItem;
+import java.util.Set;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    @Query("SELECT c FROM CartItem c WHERE c.shoppingCart.id =:shoppingCartId")
```

Comment:
such simple queries should be handled by proper method naming, please recheck work without them and fix just naming method if required